### ISSUE TITLE
Vertical Scroll

### ISSUE THREAD

**Issue by markmunoz** [cbess/AutoScrollLabel#53]
Can you add vertical scroll and multiple line support?  I am building out a tvOS app and I have a set size for a description.  I want it to fill that size and if there is more text scroll the text upwards.

**Comment by cbess** [comment]
I think I understand, you want it to scroll like it does at the end of movies, the credits?

**Comment by cbess** [comment]
Like this https://github.com/MattFoley/SWCrawlView or http://stackoverflow.com/questions/15501055/objective-%D0%A1-star-wars-opening-crawl ?

**Comment by markmunoz** [comment]
Yeah exactly like a movie credit screen.  I think Apple's own Apple TV does it for movies with long descriptions or the Plex app has done it as well for the same purpose.
